File: docs/versioned_docs/version-1.9.0/API-Reference/javascript-examples/api-projects/create-project.js
const url = `${process.env.IDRFLOW_URL ?? ""}/api/v1/projects/`;

const options = {
  method: 'POST',
  headers: {
    "Content-Type": `application/json`,
    "x-api-key": `${process.env.IDRFLOW_API_KEY ?? ""}`,
  },
  body: JSON.stringify({
  "name": "new_project_name",
  "description": "string",
  "components_list": [],
  "flows_list": []
}),
};

fetch(url, options)
  .then(async (response) => {
    if (!response.ok) {
      throw new Error(`HTTP ${response.status}`);
    }
    const text = await response.text();
    console.log(text);
  })
  .catch((error) => console.error(error));

File: docs/versioned_docs/version-1.9.0/API-Reference/python-examples/api-projects/create-project.py
import os

import requests

url = f"{os.getenv('IDRFLOW_URL', '')}/api/v1/projects/"

headers = {
    "Content-Type": "application/json",
    "x-api-key": f"{os.getenv('IDRFLOW_API_KEY', '')}",
}

payload = {"name": "new_project_name", "description": "string", "components_list": [], "flows_list": []}

response = requests.request("POST", url, headers=headers, json=payload)
response.raise_for_status()

print(response.text)
